### PR TITLE
fix: Added member data to the raw_reaction_remove event

### PR DESCRIPTION
**Pull request by jordan-day**
## Summary

This Pull Request adds the member data to the raw_reaction_remove event
This ensures user consistency between the add/remove reaction events.

## Information

- [x] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed).
- [ ] This PR is **not** a code change (e.g. documentation, README, typehinting,
      examples, ...).

## Checklist

- [x] I have searched the open pull requests for duplicates.
- [x] If code changes were made then they have been tested.
  - [N/A] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why.
- [ ] I have updated the changelog to include these changes.

### CODE REVIEW

**Comment by jordan-day** [comment]
ping @Dorukyum

**Comment by Lulalaby** [comment]
Please do not ping our team, I'm goin through and ping / DM people if needed.

**Review by Blue-Robin-Taken — APPROVED** [review]

**Comment by jordan-day** [comment]
I'd really like to get this reviewed merged if possible.
It is a minor change and will unblock my application

File: discord/state.py [hunk 1]
```diff
@@ -783,6 +783,17 @@ def parse_message_reaction_remove(self, data) -> None:
...
+        member_data = data.get("member")
+        if member_data:
+            guild = self._get_guild(raw.guild_id)
+            if guild is not None:
+                raw.member = Member(data=member_data, guild=guild, state=self)
+            else:
+                raw.member = None
+        else:
+            raw.member = None
+
```
Comment (plun1331) [inline discord/state.py:796]: This should be in the event class, not here

**Review by plun1331 — CHANGES_REQUESTED** [review]
Changelog needed

**Comment by jordan-day** [comment]
> Changelog needed

I added the changelog update.

File: discord/state.py (hunk 1 above)
Comment (jordan-day) [reply discord/state.py:796]: While I would normally agree, the code I added was taken from the `parse_message_reaction_add` function, directly above: https://github.com/jordan-day/pycord/blob/83e1bebd2c363e6dd66c45900d75a4ffde24f5f1/discord/state.py#L750

Comment (plun1331) [reply discord/state.py:796]: I can see why that was done now, thanks

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,8 @@ These changes are available on the `master` branch, but have not yet been releas
 
 ### Added
 
+- Added `member` to the `raw_reaction_remove` event.
+  ([#2412](https://github.com/Pycord-Development/pycord/pull/2412))
```
Comment (plun1331) [inline CHANGELOG.md:16]: We add changelog entries at the bottom of sections instead of the top usually, I'll fix this one for you

**Review by JustaSqu1d — CHANGES_REQUESTED** [review]
Please update the `RawReactionActionEvent.member` documentation accordingly with this change.

**Comment by jordan-day** [comment]
> Please update the `RawReactionActionEvent.member` documentation accordingly with this change.

Done

**Review by Lulalaby — APPROVED** [review]

**Review by Lulalaby — APPROVED** [review]

**Review by Dorukyum — APPROVED** [review]
LGTM

**Review by Lulalaby — APPROVED** [review]
pullapprove do your job

**Pull request merged**